db/migrate/20150828132451_remove_duplicate_and_empty_pods.rb

```diff
@@ -0,0 +1,28 @@
+class RemoveDuplicateAndEmptyPods < ActiveRecord::Migration
+  def up
+    remove_dupes
+    remove_empty_or_nil
+
+    add_index :pods, :host, unique: true, length: 190 # 190*4 for utf8bin
+  end
+
+  def down
+    remove_index :pods, :host
+  end
+
+  private
+
+  def remove_dupes
+    duplicates = Pod.group(:host).count.select { |k, v| v > 1 }.keys
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.